### DISCUSSION TITLE
The youth

### DISCUSSION THREAD

**Discussion by brh** [Adventech/sabbath-school-android#755] [General]
Will there be a focus on the youth lesson? Especially the teens and college age (Realtime faith lessons and up), they are very likely to be ones to be using devices and they need the focus just as much if not more. The Powerpoint lessons are graphic heavy and have some word puzzles that may make it prohibitive to develop.

I know there might need to be some changes at the conference level (Realtime faith needs an update format wise, probably needs to fall in line with Cornerstone level), do you have a contact there, know of any plans?

**Comment by VitalikL** [comment]
Closing as this has been added a while ago.

**Reply by brh** [reply]
I will reopen this, the Cornerstone lesson is just a pdf view or whatever.  I am talking about an actual lesson where they could type in answers and highlight stuff like the adult lesson.
